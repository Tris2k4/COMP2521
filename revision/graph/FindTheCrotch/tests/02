10
11
6-8 6-3 8-3 3-4 7-4 4-9 9-0 9-2