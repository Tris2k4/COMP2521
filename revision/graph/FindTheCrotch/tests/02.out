Enter number of vertices: Enter number of edges: Enter edges in the form v-w: 
Graph: nV = 10
Edges:
 0: 0-9
 1:
 2: 2-9
 3: 3-4 3-6 3-8
 4: 4-3 4-7 4-9
 5:
 6: 6-3 6-8
 7: 7-4
 8: 8-3 8-6
 9: 9-0 9-2 9-4

The crotch is at vertex 9.
